spec/rubocop/cop/rspec/context_wording_spec.rb

```diff
@@ -33,6 +33,13 @@
     RUBY
   end
 
+  it "skips descriptions beginning with 'for'" do
```

Comment:
Since this a configuration change rather than behaviour, I don't think an additional test is necessary.